kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1932,6 +1932,17 @@ object KyuubiConf {
       .stringConf
       .createWithDefault("server_operation_logs")
 
+  val PROXY_USER: OptionalConfigEntry[String] =
+    buildConf("kyuubi.proxy.user")
+      .doc("An alternative to hive.server2.proxy.user. " +
+        "The current behavior is consistent with hive.server2.proxy.user " +
+        "and takes effect only in FrontendServices, " +
+        "which contains TFrontendService and KyuubiRestFrontendService. " +
```

Comment:
a general tip: docs are for end users, we should say it affects on RESTful API instead of `KyuubiRestFrontendService`